code/game/objects/items/weapons/grenades/atmosgrenade.dm

```diff
@@ -18,7 +18,33 @@
 		target_turf.air_update_turf()
 	qdel(src)
 
+/obj/item/grenade/gas/plasma
+	icon_state = "plasma"
+
 /obj/item/grenade/gas/knockout
-	name = "Knockout Grenade"
+	name = "knockout grenade"
 	desc = "A grenade that completely removes all air and heat from its detonation area."
-	spawn_contents = SPAWN_20C | SPAWN_N2O
\ No newline at end of file
+	spawn_contents = SPAWN_20C | SPAWN_N2O
+
+/obj/item/grenade/gluon
+	desc = "An advanced grenade that releases a harmful stream of gluons inducing radiation in those nearby. These gluon streams will also make victims feel exhausted, and induce shivering. This extreme coldness will also wet any nearby floors."
+	name = "gluon grenade"
+	icon = 'icons/obj/grenade.dmi'
+	icon_state = "gluon"
+	item_state = "flashbang"
+	var/range = 4
+	var/rad_damage = 60
+	var/stamina_damage = 30
+
+/obj/item/grenade/gluon/prime()
+	update_mob()
+	playsound(loc, 'sound/effects/empulse.ogg', 50, 1)
+	for(var/turf/T in view(range,loc))
+		if(isfloorturf(T))
+			var/turf/simulated/F = T
+			F.MakeSlippery(TURF_WET_PERMAFROST)
+			for(var/mob/living/carbon/L in T)
+				L.adjustStaminaLoss(stamina_damage)
+				L.apply_effect(rad_damage, IRRADIATE)
+				L.adjust_bodytemperature(-230)
```

Comment:
This should probably have a `min()` proc attached so that the mob's body temperature can't go below 0K

Reply:
This is handled by the adjust_bodytemperature proc.